generators/common/generator.mjs

```diff
@@ -124,15 +132,15 @@ export default class CommonGenerator extends BaseApplicationGenerator {
     return {
       setupConstants({ application }) {
         // Make constants available in templates
-        application.MAIN_DIR = constants.MAIN_DIR;
-        application.TEST_DIR = constants.TEST_DIR;
-        application.SERVER_MAIN_RES_DIR = constants.SERVER_MAIN_RES_DIR;
-        application.ANGULAR = constants.SUPPORTED_CLIENT_FRAMEWORKS.ANGULAR;
-        application.REACT = constants.SUPPORTED_CLIENT_FRAMEWORKS.REACT;
+        application.MAIN_DIR = MAIN_DIR;
+        application.TEST_DIR = TEST_DIR;
+        application.SERVER_MAIN_RES_DIR = SERVER_MAIN_RES_DIR;
+        application.ANGULAR = ANGULAR;
+        application.REACT = REACT;
```

Comment:
Will not be exported to clients in the future.
Should be variables.
```suggestion
```

Reply:
Fully agree :-)